### ISSUE TITLE
Add 5.10 kernel

### ISSUE THREAD

**Issue by mobilegmYT** [dhruvvyas90/qemu-rpi-kernel#127]
Recently the RPI foundation bumped the kernels to v5.10, it would be nice as well to have a updated kernel file for running newer RPI OS versions in QEMU.

**Comment by ElieLiabeuf** [comment]
Also interested.

Can't understand why it's working fine with the kernel8 provided here : https://github.com/dhruvvyas90/qemu-rpi-kernel/tree/master/native-emuation but if I follow the instruction and extract kernel8 from the last raspios it doesnt boot...

**Comment by NWJeff** [comment]
> 
> 
> Also interested.
> 
> Can't understand why it's working fine with the kernel8 provided here : https://github.com/dhruvvyas90/qemu-rpi-kernel/tree/master/native-emuation but if I follow the instruction and extract kernel8 from the last raspios it doesnt boot...

I have this same issue with the kernel8 that I extracted. Has anyone had success with the new kernel and QEMU? Any suggestions for troubleshooting? May not be an issue that  dhruvvyas90 can fix. Thank you.

**Comment by ElieLiabeuf** [comment]
> I have this same issue with the kernel8 that I extracted. Has anyone had success with the new kernel and QEMU? Any suggestions for troubleshooting? May not be an issue that dhruvvyas90 can fix. Thank you.

Yep, to me it's related to QEMU. If you take the last raspios using kernel 5.4.83 ([this one](https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2021-01-12/2021-01-11-raspios-buster-armhf-lite.zip)) it works just fine. You can extract bcm2710-rpi-3-b-plus.dtb and kernel8.img and everything runs.

**Comment by mobilegmYT** [comment]
Maybe we could use https://github.com/dhruvvyas90/qemu-rpi-kernel/tree/master/tools?

**Comment by mobilegmYT** [comment]
> You can extract bcm2710-rpi-3-b-plus.dtb and kernel8.img and everything runs.

What did you mean by this? Like if I extract these out and point qemu to use these instead of the dtb in this repo?

**Comment by ptrsr** [comment]
@mobilegmYT right. I can also confirm that `5.4.83` works when compiled from source, while `5.10` does not boot. Might try some different kernel and QEMU versions in the future.

**Comment by OrangeDog** [comment]
This was done: 8f85476768cc309ff1d46ed10518a5e2a1f08058
